File: c course/c w3resouse questons/q22.c
#include<stdio.h>
int main(){
    int num[5],i,total=0;
    printf("enter number 1\n");
    scanf("%d",&num[0]);

    printf("enter number 2\n");
    scanf("%d",&num[1]);

    printf("enter number 3\n");
    scanf("%d",&num[2]);

    printf("enter number 4\n");
    scanf("%d",&num[3]);

    printf("enter number 5\n");
    scanf("%d",&num[4]);

    for(i=0;i<5;i++)
    {
        if((num[i]%2)!=0)
        {
            total +=num[i];
        }
    }
    printf("sum of all odd number :%d",total);

return 0;
}

File: Python/geeks for geek/list/q22.py

lst=[1,2,3,4,5,6,7,8,9,10,11]
count_even,count_odd=0,0

for i in lst:
    if i%2==0:
        count_even+=1
    elif i%2==1:
        count_odd+=1

print("Total even number in the list : ",count_even)
print("Total odd number in the list : ",count_odd)
